rasa/server.py

```diff
@@ -567,6 +567,13 @@ def create_app(
                 {"parameter": "name", "in": "body"},
             )
 
+        # Deprecation warning
+        warnings.warn(
+            "Triggering actions via the execute endpoint is deprecated. "
+            "Trigger an intent via the inject-intent endpoint instead.",
```

Comment:
I know that the old endpoint and the new one are quite similar, but maybe we could replace `inject-intent` with a more full specification like `/conversations/<conversation_id>/inject-intent`